src/ifElse.test.js

```diff
@@ -1,11 +1,72 @@
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
 
   it('should ', () => {
 
   });
 
-  // write tests here
+  it('should call the first function when the condition is true', () => {
+    const condition = jest.fn().mockReturnValue(true);
+    const first = jest.fn();
+    const second = jest.fn();
```

Comment:
what are the 'first' and 'second'? give the variables understandable names
for example firstCallback and secondCallback
fix it everywehre